definitions/json/AuthorisationCaseField/AuthorisationCaseField-Bundles.json

```diff
@@ -556,5 +556,29 @@
     "CaseFieldID": "bundlesTabTitle",
     "UserRole": "[SOLICITORJ]",
     "CRUD": "CRU"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "TTL",
+    "UserRole": "TTL_profile",
+    "CRUD": "CRU"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "TTL",
+    "UserRole": "caseworker-employment-englandwales",
+    "CRUD": "CRU"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "TTL",
+    "UserRole": "citizen",
+    "CRUD": "CRU"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "TTL",
+    "UserRole": "caseworker-employment-api",
```

Comment:
Do we also need this for legal reps as they can also create draft cases

Reply:
added.